cmd/kwil-cli/cmds/database/call.go

```diff
@@ -102,6 +111,7 @@ func callCmd() *cobra.Command {
 	cmd.Flags().StringP(dbidFlag, "i", "", "the target database id")
 	cmd.Flags().StringVarP(&action, actionNameFlag, "a", "", "the target action name (required)")
 	cmd.Flags().BoolVar(&authenticate, "authenticate", false, "authenticate signals that the call is being made to a gateway and should be authenticated with the private key")
+	cmd.Flags().BoolVarP(&requestChallenge, "challenge", "c", false, "request a challenge from the server to authenticate the call")
```

Comment:
I find this flag's name and description to be a little too implementation oriented.  The point here is that we are indicating that we expect to be making a call to a server that requires a signature in the call request so server can authenticate us.  Using a challenge is a detail of our approach to preventing replay.
Can this work another name like `auth` or `signed`?  I know it's confusign with the gateway's `authenticate`, but I don't think `challenge` is to the point.